grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -1111,6 +1112,74 @@ class IndicatorDataService {
         return graphData.toJson()
     }
 
+    Map getBackdatedShipmentsChart(String query, String locationId, Integer monthsLimit) {
+        Map<String, Integer> data = [
+                '2 days': 0,
+                '3 days': 0,
+                '4 days': 0,
+                '5 days': 0,
+                '6 days': 0,
+                '7 days': 0,
+                '7+ days': 0,
```

Comment:
I would suggest moving those "2 days" etc. to static constants in `Constants.groovy`, as this might be reused in the future.